cirq/google/engine/engine_test.py

```diff
@@ -14,6 +14,7 @@
 
 """Tests for engine."""
 import base64
+import copy
```

Comment:
not sure why this was added

Reply:
I added this for json.dumps for generating the error content, but I can just make a string. \Removed.